.editorconfig

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2022-present, IO Visor Project
+# SPDX-License-Identifier: Apache-2.0
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = crlf
```

Comment:
Can we remove this line `end_of_line = crlf`?  This will cause problems on macOS/Linux.

We should rely on Git to handle line termination on different platforms when committing and checking code out.